app/models/train.rb

```diff
@@ -127,6 +128,7 @@ class Train < ApplicationRecord
   before_update :disable_copy_approvals, unless: :approvals_enabled?
   after_update :schedule_release!, if: -> { kickoff_at.present? && kickoff_at_previously_was.blank? }
   after_update :create_default_notification_settings, if: -> { notification_channel.present? && notification_channel_previously_was.blank? }
+  after_update :restore_default_notification_settings, if: -> { notification_channel.present? && notifications_release_specific_channel_enabled_previously_was == true }
```

Comment:
This is a good catch, but we need to check that it's `false` now as well, not just that it was `true` before?